contract_manager/scripts/send_message_to_wormhole.ts

```diff
@@ -0,0 +1,55 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { WormholeEmitter, loadHotWallet } from "../src/governance";
+
+const parser = yargs(hideBin(process.argv))
+  .usage(
+    "Usage: $0 --cluster <cluster> --wallet-path <path_to_wallet_file> --message <message>"
+  )
+  .options({
+    cluster: {
+      type: "string",
+      demandOption: true,
```

Comment:
I this this is gonna only handle `mainnet-beta` and `testnet`. It's probably a good idea to add these as options here for better type hints and more verification